TransifexNativeSDK/clitool/src/main/java/com/transifex/clitool/MainClass.java

```diff
@@ -165,6 +171,16 @@ public Integer call() throws Exception {
                 }
             }
 
+            // Append custom tags
+            if (tags != null && tags.length != 0) {
+                for (LocaleData.StringInfo stringInfo : sourceStringMap.values()) {
+                    if (stringInfo.meta == null) {
+                        stringInfo.meta = new LocaleData.StringInfo.Meta();
+                    }
+                    stringInfo.meta.tags = tags;
+                }
+            }
```

Comment:
Is there a reason we're not always adding the `Meta`? It would make more sense to me to have it always and add the `if` just before setting `stringInfo.meta.tags = tags;`.

Reply:
At this moment in time, `Meta` only contains `tags` and tags can't be parsed by the XML converter (in the previous step). So yeah, we could always add it since it's always null.

But in the future, the `Meta` could also contain developer comments or other stuff (which are supported by the spec). So, why overwrite it since the XML Parser could have populated it?